src/metakb/transform/moa.py

```diff
@@ -148,14 +157,14 @@ async def _add_variant_therapeutic_response_studies(
                     # skipping HORMONE and CHEMOTHERAPY for now
                     continue
 
-                therapies = [tn.strip() for tn in therapy_name.split("+")]
+                therapies = [{"label": tn.strip()} for tn in therapy_name.split("+")]
```

Comment:
I think I may be missing the reason for using a dict, can you explain this change?

Reply:
yeah -- in MOA, `get_therapeutic_agent` had broken the interface [defined in the abstract class](https://github.com/cancervariants/metakb/blob/836f47d63bebbea31708735e62ca80d4a468caf9/src/metakb/transform/base.py#L302) by making the argument a string. I assume this is because in CIViC, drugs are more richly described, but they're just a single name in MOA. I think the simplest way to make a generalization that's true for both is to just have the MOA drugs just be simple dicts with a single key/value.

Reply:
Sounds good! Thanks